test/integration/command-monitoring/command_monitoring.spec.test.js

```diff
@@ -278,18 +280,6 @@ describe('Command Monitoring spec tests', function () {
   });
 
   describe('command monitoring unified spec tests', () => {
-    for (const loadedSpec of loadSpecTests('command-monitoring/unified')) {
-      expect(loadedSpec).to.include.all.keys(['description', 'tests']);
-      context(String(loadedSpec.description), function () {
-        for (const test of loadedSpec.tests) {
-          it(String(test.description), {
-            metadata: { sessions: { skipLeakTests: true } },
-            test: async function () {
-              await runUnifiedTest(this, loadedSpec, test);
-            }
-          });
-        }
-      });
-    }
+    runUnifiedSuite(loadSpecTests(path.join('command-monitoring', 'unified')));
```

Comment:
just curious, is there any benefit to using `path.join` in place of `command-monitoring/unified` as a string?

Reply:
this used to be best practice for specifying paths because of how paths resolved on windows vs linux systems, I honestly don't know if the underlying logic here works automagically nowadays to make them resolve fine on windows even with the forward slash

Reply:
it looks like loadspectests feeds it into a path.resolve, so that also normalizes it, which means we could get away with not using path.join, but I feel like this is one of those better safe than sorry things

Reply:
Windows supports both directions of slash these days, but you never know! I appreciate a cross platform abstraction when I see one.